local-tests/tests/testEthereumBroadcastTransactionWrappedKey.ts

```diff
@@ -40,6 +37,8 @@ export const testEthereumBroadcastTransactionWrappedKey = async (
     pkpSessionSigs,
     privateKey: wrappedKeysWalletPrivateKey,
     litNodeClient: devEnv.litNodeClient,
+    publicKey: '0xdeadbeef',
```

Comment:
Yup -- it's not actually used for anything, and these are actually writing to Dynamodb and are only used for integration testing the API.  We don't validate the structure of the publicKey on the backend, so I set the integration tests to write this string so that we could safely clean out the DynamoDB backend of these records intermittently.

A better solution here would be to set up a CI instance of the backend wrapped keys API instead, so we can pave it over regularly and not spend $$$$$ on storing (eventually) thousands and millions of temporary keys that our CI and tests generate ;).

Reply:
Interesting idea but for now just keeping the same publicKey seems easier and we can just periodically delete it from the DB